DNN Platform/Tests/DotNetNuke.Tests.Web/UrlRewrite/RewriteControllerUnitTests.cs

```diff
@@ -0,0 +1,21 @@
+﻿using DotNetNuke.Entities.Urls;
+using NUnit.Framework;
+
+namespace DotNetNuke.Tests.Web.UrlRewrite
+{
+    [TestFixture]
+    class RewriteControllerUnitTests
+    {
+        [TestCase("http://www.dnn.com/en-us/aboutus", "en-us", "http://www.dnn.com/aboutus")]
+        [TestCase("http://www.dnn.com/en-Us/aboutus", "en-Us", "http://www.dnn.com/aboutus")]
+        [TestCase("http://www.dnn.com/aboutus", "en-Us", "http://www.dnn.com/aboutus")]
+        public void RemoveLanguageCodeFromAbsoluteURL(string absoluteURL, string languageCode, string expectedURL)
```

Comment:
@mitchelsellers @sleupold  I'm not able to understand the breaking changes please guide by pointing out the missing case